### DISCUSSION TITLE
How import Postman JSON collection?

### DISCUSSION THREAD

**Discussion by boardtc** [hoppscotch/hoppscotch#2168] [Ideas]
I have a collection exported from Postman in JSON. is there any plans to create an import or any workaround to import currently?

**Comment by AndrewBastin** [comment]
@boardtc you can use the button over here to import from Postman, Insomnia and OpenAPI.

![image](https://user-images.githubusercontent.com/9131943/158383242-6212612d-c3c7-4fb0-8e81-a778330ccfe6.png)

@liyasthomas proper docs for this or something ? ux analysis ?

**Comment by boardtc** [comment]
Note, I searched the docs for import before I posted, there is nothing there.

I found _My Collections_ as a right-hand sidebar under the 3rd icon on the left sidebar, _Docs_ and imported. Great. 

Then however the My Collections sidebar disappeared when docs were selected....not sure how to get it back...

And also how can I make the My Collections sidebar visible when I have REST selected?

**Reply by boardtc** [reply]
After 15 minutes today I found the collapse sidebar button in the bottom corner!!
And the option in the picture above :-)
Phew!!
Just the help needs updating so.

**Reply by AndrewBastin** [reply]
Oh gotcha. We will try to improve that experience in the future. Thanks for sticking on ❤️ 

@liyasthomas 👀

**Comment by mikbagautdinov** [comment]
Hello! I have a GraphQL collection imported from Postman. Is there any way to add it to hoppscotch? Even though I managed to do it in REST section, it doesn't work as it's GraphQL collection.( I would appreciate any answer.
